scripts/recording_uploader/uploader/app.py

```diff
@@ -0,0 +1,40 @@
+"""Lambda function for generating a presigned URL for uploading a recording to S3."""
+
+from typing import Any
+from uuid import uuid4
+import json
+
+from botocore.client import Config
+import boto3
+
+
+def lambda_handler(*args: Any, **kwargs: Any) -> dict:
+    """Main entry point for the lambda function."""
+    return {
+        "statusCode": 200,
+        "body": json.dumps(get_presigned_url()),
+    }
+
+
+def get_presigned_url() -> dict:
+    """Generate a presigned URL for uploading a recording to S3."""
+    bucket = "openadapt"
+    region_name = "us-east-1"
```

Comment:
What do you think about putting these in `NAMED_CONSTANTS` at the top of this file, and passing them in as default keyword arguments to `get_presigned_url`?